src/components/forms/SearchBar/SearchSuggestions.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { isNotNull } from 'ramda-adjunct';
+
+import { getDepth } from '../../../utils/helpers';
+import { useOuterClick } from '../../Datatable/hooks/useOuterCallback';
+import {
+  DropdownLink,
+  List,
+  StyledDropdownPane,
+} from '../../Datatable/Dropdown/DropdownPane';
+import {
+  SearchSuggestionPropType,
+  SearchSuggestionsPaneProps,
+} from './SearchSuggestions.types';
+import { SearchSuggestionFormats } from './Search.enums';
+
+const SearchSuggestionsPane = styled(StyledDropdownPane)`
+  min-width: 100%;
+  z-index: ${getDepth('dropdown')};
+`;
+
+const SearchSuggestionsLink = styled(DropdownLink)`
+  width: 100%;
+`;
+
+const SearchSuggestions: React.FC<SearchSuggestionsPaneProps> = ({
+  suggestions,
+  searchSuggestionFormat = 'default',
+  onClickOut,
+}) => {
+  const searchSuggestionsPaneRef = useOuterClick(onClickOut);
+  const searchSuggestion = SearchSuggestionFormats[searchSuggestionFormat];
+
+  return (
+    <SearchSuggestionsPane ref={searchSuggestionsPaneRef}>
+      <List>
+        {suggestions.map((suggestion) => {
+          return (
+            isNotNull(suggestion) && (
+              <li key={suggestion.name}>
```

Comment:
I missing posibility to pass custom suggestion renderer as we were talking during the grooming. We need to be able pass custom component which will be rendered here.
So there should be `suggestionRenderer` prop which accepts function which have `suggestion` argument and returns JSX.element. Basically same as custom renderers in `react-table`.
And by default it will render just a string.

Reply:
fixed by adding `renderSearchSuggestion`